testutil-client/src/test/java/io/spine/client/blackbox/AcknowledgementsTest.java

```diff
@@ -48,9 +48,9 @@
 import static io.spine.client.blackbox.given.CommandAcksTestEnv.projectAlreadyStarted;
 import static io.spine.client.blackbox.given.CommandAcksTestEnv.taskCreatedInCompletedProject;
 import static io.spine.client.blackbox.given.CommandAcksTestEnv.taskLimitReached;
-import static io.spine.testing.client.blackbox.Rejections.BbProjectAlreadyStarted;
-import static io.spine.testing.client.blackbox.Rejections.BbTaskCreatedInCompletedProject;
-import static io.spine.testing.client.blackbox.Rejections.BbTaskLimitReached;
+import static io.spine.test.testutil.blackbox.Rejections.BbProjectAlreadyStarted;
```

Comment:
@mdrachuk , @armiol , what is `testutil` in this context? 

I understand, classes under `test` are test data, but what kinds of utilities we have under `test`?
If they are really utilities, why aren't they parts of `testing.server` or `testing.client` according to our new naming conventions?

Reply:
@mdrachuk @alexander-yevsyukov as [discussed previously](https://github.com/SpineEventEngine/core-java/pull/740#discussion_r201697877), the repackaging of `testutil-*` modules is currently in progress. This package naming will be changed in scope of the repackaging in separate PRs.

Reply:
@vlad-lubenskyi is currently working on repackaging existing test-utils, so I’ve set them to match an existing convention to avoid any confusion.